resolwe_bio/processes/expression/featureCounts.yml

```diff
@@ -42,14 +42,40 @@
             Indicate if strand-specific read counting should be performed. For
             paired-end reads, strand of the first read is taken as the strand
             of the whole fragment. FLAG field is used to tell if a read is
-            first or second read in a pair.
+            first or second read in a pair. Automated strand detection is enabled
+            using the [Salmon](https://salmon.readthedocs.io/en/latest/library_type.html)
+            tool's build-in functionality. To use this option, cDNA (transcriptome)
+            index file crated using the Salmon indexing tool must be provided.
           choices:
             - label: Strand non-specific
               value: non_specific
             - label: Strand-specific forward
               value: forward
             - label: Strand-specific reverse
               value: reverse
+            - label: Detect automatically
+              value: auto
+        - name: cdna_index
+          label: cDNA index file
+          type: data:index:salmon
+          required: false
+          hidden: "alignment.assay_type != 'auto'"
+          description: >
+            Transcriptome index file created using the Salmon indexing tool.
+            cDNA (transcriptome) sequences used for index file creation must be
+            derived from the same species as the input sequencing reads to
+            obtain the reliable analysis results.
+        - name: sampling_rate
+          label: Alignment file sampling rate
+          type: basic:decimal
+          default: 0.01
+          required: false
```

Comment:
Setting the `required` attribute is redundant, if `default` is used. The same applies also to the workflow and other associated processes.